### PR TITLE
Mijke jetten patch 2

### PR DESCRIPTION
**Pull request by MijkeJetten**
Updated Why FAIR lesson plan (outcomes, activities, overview, prerequisites + cleaned up)

### CODE REVIEW

**Comment by MijkeJetten** [comment]
Hi @martijnkersloot @bianchini88 hope my changes didn't break anything. Can you please check if all code is ok and both tables are ok?

**Comment by martijnkersloot** [comment]
@MijkeJetten Thanks for the heads up :).  I'll update your branch to follow the new structure!

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 1]
```diff
@@ -8,56 +8,143 @@ authors:
...
+      level: intermediate
+
+    - learning_outcome: 4
+      activities: Divide participants into groups to identify and share how each stakeholder’s needs align with FAIR principles in a research project
+      time: 20 min
+      type: group discussion
+      level: intermediate
+      
+    - learning_outcome: 4
+      activities: Organize a workshop where participants evaluate a real or hypothetical project’s current practices, then develop a detailed action plan to implement FAIR principles
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:113]:
```suggestion
      activities: Organise a workshop where participants evaluate a real or hypothetical project’s current practices, then develop a detailed action plan to implement the FAIR principles
```

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 2]
```diff
@@ -8,56 +8,143 @@ authors:
...
+    - learning_outcome: 3
+      activities: Have participants list common issues in research that arise from non-FAIR data practices, such as data loss or inaccessibility
+      time: 10 min
+      type: group discussion
       level: beginner
 
-    - learning_outcome: 6
-      activities: For each stakeholder identified in the previous exercise, go through each letter of FAIR and explore why they should be motivated to implement each principle.
+    - learning_outcome: 3
+      activities: Ask participants to analyze case studies of projects that failed due to non-FAIR practices and discuss the repercussions
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:89]:
```suggestion
      activities: Ask participants to analyse case studies of projects that failed due to non-FAIR practices and discuss the repercussions
```

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 3]
```diff
@@ -8,56 +8,143 @@ authors:
...
+
+    - learning_outcome: 4
+      activities: Introduce participants to key changes needed to adopt FAIR principles within a research team, using simple examples
       time: 10 min
-      type: discussion
+      type: group discussion
       level: beginner
 
+    - learning_outcome: 4
+      activities: In groups, participants analyze a project scenario and identify specific changes needed to meet FAIR principles, sharing their findings
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:101]:
```suggestion
      activities: In groups, participants analyse a project scenario and identify specific changes needed to meet FAIR principles, sharing their findings
```

File: docs/units/FAIR-generics/topics/1-why-fair.md (hunk 1 above)
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:113]:
```suggestion
      activities: Organise a workshop where participants evaluate a real or hypothetical project’s current practices, then develop a detailed action plan to implement FAIR principles
```

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 4]
```diff
@@ -115,7 +202,7 @@ references:
   - title: Ten reasons to share your data
     author: Amanda S. Barnard
     url: https://www.natureindex.com/news-blog/ten-reasons-to-share-your-data
-  - title: "FAIR Cookbook: [What are the FAIR Principles?"
+  - title: FAIR Cookbook: What are the FAIR Principles?
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:205]: I am not sure whether the ":" can cause issues. They did sometimes in the old version.

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 5]
```diff
@@ -125,7 +212,7 @@ references:
   - title: The FAIR principles of data management
     url: https://genestack.com/resources/library/the-fair-principles-of-data-management/
     additionalInformation: section about stakeholders
-  - title: "Implementing FAIR in data sharing: who are the stakeholders and what are their responsibilities?"
+  - title: Implementing FAIR in data sharing: who are the stakeholders and what are their responsibilities?
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:215]: As above, check if the ":" causes issues here.

**Review by bianchini88 — CHANGES_REQUESTED** [review]
Some minor comments mostly style/spelling related.

File: docs/units/FAIR-generics/topics/1-why-fair.md (hunk 4 above)
Comment (MijkeJetten) [reply docs/units/FAIR-generics/topics/1-why-fair.md:205]: Maybe @martijnkersloot knows (and can adjust if necessary)

File: docs/units/FAIR-generics/topics/1-why-fair.md (hunk 5 above)
Comment (MijkeJetten) [reply docs/units/FAIR-generics/topics/1-why-fair.md:215]: Maybe @martijnkersloot knows (and can adjust if necessary)

**Comment by MijkeJetten** [comment]
@martijnkersloot @bianchini88 if we can deploy before Tuesday (as I will be giving the presentation then) that would be great

File: docs/units/FAIR-generics/topics/1-why-fair.md (hunk 4 above)
Comment (bianchini88) [reply docs/units/FAIR-generics/topics/1-why-fair.md]: I verified they did indeed cause errors. I removed them (see commits below).

**Comment by bianchini88** [comment]
@MijkeJetten @martijnkersloot I solved merging conflicts and addressed some other minor issues. I think it should be mergeable now. It requires the approval of somebody other than me.

**Comment by MijkeJetten** [comment]
@martijnkersloot @konradana @nrvanulzen please one of you approve, I can't

**Review by konradana — APPROVED** [review]

**Comment by bianchini88** [comment]
@MijkeJetten @martijnkersloot @konradana @nrvanulzen I have issues previewing the webpage locally. Does any of you manage to preview the page locally? Or do we merge the PR and take it from there?

**Comment by MijkeJetten** [comment]
I didn't manage to view it locally. Can we merge and take it from there? Or is that too tricky?

**Comment by MijkeJetten** [comment]
I don't think that is an issue now. @martijnkersloot agree? It would be great if we can fix it ASAP though haha, since I need to present tomorrow

**Comment by martijnkersloot** [comment]
Yep, I'll check it out on my own computer as I think that some things will not work yet. I'll ping you later today!

**Comment by bianchini88** [comment]
@martijnkersloot Can you share the Gemfile.lock and the Gemfile (if you have to edit it to make it work)? 
I need to add something like  `gem "faraday", "< 1.0` to my Gemfile to make it work and have to use 
`gem install bundler -v 2.4.22`. This approach does not crash, but things are not visualised properly...

**Pull request merged**